package.json

```diff
@@ -6,23 +6,21 @@
   "author": "StackStorm <support@stackstorm.com>",
   "description": "A simple helpful robot for your Company, with StackStorm Support",
   "dependencies": {
-    "hubot"             : "^2.19.0",
+    "hubot"             : "^3.1.1",
     "hubot-diagnostics" : "0.0.2",
     "hubot-redis-brain" : "0.0.4",
     "hubot-scripts"     : "^2.17.2",
     "hubot-help"        : "^0.2.2",
-    "hubot-stackstorm"  : "^0.8.0",
-    "hubot-flowdock"    : "^0.7.6",
-    "hubot-matteruser"  : "^3.6.0",
-    "hubot-yammer"      : "git+https://github.com/athieriot/hubot-yammer",
-    "hubot-spark"       : "^1.7.0",
+    "hubot-stackstorm"  : "git+https://github.com/StackStorm/hubot-stackstorm#update-components",
+    "hubot-flowdock"    : "^0.7.8",
+    "hubot-matteruser"  : "^4.4.0",
+    "hubot-spark"       : "^2.0.0",
     "hubot-xmpp"        : "^0.2.5",
-    "hubot-hipchat"     : "git+https://github.com/StackStorm/hubot-hipchat#sonnyp-patch",
```

Comment:
That's what I would prefer - rather than removing it completely right now, provide notification that we're going to remove it from the core packages in the near future. 

We should also make sure that we provide docs for users on how to re-add those adapters if they want to continue using them.